apps/web/pages/settings/developer.tsx

```diff
@@ -0,0 +1,19 @@
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import ApiKeyListContainer from "@ee/components/apiKeys/ApiKeyListContainer";
+
+import SettingsShell from "@components/SettingsShell";
+import Shell from "@components/Shell";
+import WebhookListContainer from "@components/webhook/WebhookListContainer";
+
+export default function Settings() {
+  const { t } = useLocale();
+
+  return (
+    <Shell heading={t("developer")} subtitle={t("manage_developer_settings")}>
+      <SettingsShell>
+        <WebhookListContainer title={t("webhooks")} subtitle={t("receive_cal_meeting_data")} />
+        <ApiKeyListContainer />
```

Comment:
As mentioned in the chat, and approved by Ciarán, API Keys live in Developers tab in Settings now